### DISCUSSION TITLE
Prompt in dmenu with fancy theme

### DISCUSSION THREAD

**Discussion by kaistian** [davatorium/rofi#1359] [Q&A] [answered]
I really like the fancy theme, but when I try to use a prompt in dmenu mode I don't get the prompt.
Is this by the design or is it something missing in the fancy theme?

This is the command I run:
`echo -e "no\nyes" | rofi -no-config -dmenu -p Delete -theme /usr/share/rofi/themes/fancy.rasi`

**Comment by DaveDavenport** [comment, marked as answer]
The fancy theme does not have a prompt. So it is expected to not show up.

**Reply by kaistian** [reply]
Thanks.
I'll start looking into themes so I can add one for my use case.
